libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1751,6 +1765,14 @@ void AP_InertialSensor::acal_update()
     }
 }
 
+// Update the harmonic notch frequency
+void AP_InertialSensor::update_harmonic_notch_freq_hz(float scaled_freq) {
+    // When disarmed, throttle is zero
```

Comment:
this is a bit of a funny comment here I think.. copy-paste issue?
Also I don't think that this method, update_harmonic_notch_freq_hz, is ever called so I think it would be best to remove it for now.

Reply:
So the issue here is that we want to add the harmonic update PR immediately after this and if we remove it here it means AP_InertialSensor is in that PR and review. I guess the thinking from @lthall and I was get all the difficult reviews out the way here and then the follow-on PR becomes much simpler. Or we add that code in here. Or we leave it as is. So your call from a process perspective. Also the update code is more naturally associated with the harmonic filter code.

Reply:
if it's coming in the next few weeks then OK.. if it's longer than that then let's put it in the next PR..

Reply:
Cool - it's ready to go. I have been flying with it for several weeks now. So I'll leave this in.

Reply:
Changed the comment